### ISSUE TITLE
final review - xt222

### ISSUE THREAD

**Issue by XinyunTang** [nehalrawat/PredictingMedicalCosts#13]
The goal of the project is to predict how much hospital in the Bronx bill patients coming in with chest pains using a series of features. 

The things I like: 1) the report is very clear when describing the features and feature engineering techniques they used. 2) They tried various models from the class and compared their prediction accuracy throughout the project.

For improvements: 1) Did you guys use the same training and test set for all the models you ran? If you didn't use the training and testing set, it's meaningless to compare their residual error. 2) how do you choose the hyperparameters in these models? Is there a specific reason why you choose a certain parameter? If there is no sufficient reason, maybe you should use crossvalidation to see which one does better (e.g. the quantile used in quantile regression). 3) I don't think you have convinced me that only looking at bronx data is necessary. I think you should use data from the entire NYC, in which case you have a much larger dataset. 

Generally, I think the group did OK but there certainly are a lot of improvements that can be done.
